src/input_common/drivers/mouse.cpp

```diff
@@ -171,13 +176,39 @@ AnalogMapping Mouse::GetAnalogMappingForDevice(
     return mapping;
 }
 
+Common::Input::ButtonNames Mouse::GetUIButtonName(const Common::ParamPackage& params) const {
+    MouseButton button = static_cast<MouseButton>(params.Get("button", 0));
```

Comment:
```suggestion
    const auto button = static_cast<MouseButton>(params.Get("button", 0));
```

since the type is already on the right hand side.